.github/workflows/CI_build.yml

```diff
@@ -33,6 +33,7 @@ jobs:
     - name: nmake libcurl
       working-directory: curl/winbuild
       run: |
+           set WINBUILD_ACKNOWLEDGE_DEPRECATED=yes
            nmake /f Makefile.vc mode=dll vc=15 RTLIBCFG=static           MACHINE="${{ matrix.build_platform }}"
            nmake /f Makefile.vc mode=dll vc=15 RTLIBCFG=static DEBUG=yes MACHINE="${{ matrix.build_platform }}"
```

Comment:
@donho Just add it here and in the line above like `RTLIBCFG=static` or `DEBUG=yes`

Reply:
@donho @chcg 
But then, the "set" in line 36 is not needed.